pkg/crc/oc/oc.go

```diff
@@ -82,3 +85,39 @@ func (oc OcConfig) RunOcCommandPrivate(args ...string) (string, string, error) {
 	}
 	return oc.Runner.RunPrivate(args...)
 }
+
+type OcVMConfig struct {
+	Runner  *ssh.SSHRunner
+	Context string
+	Cluster string
+}
+
+func UseOCWithVMConfig(sshRunner *ssh.SSHRunner) OcVMConfig {
```

Comment:
You can avoid to define this new type by creating a new Runner (and maybe remove use of stderr?).

``` 
type OcSSHRunner struct {
	Runner *ssh.SSHRunner
}

func (r OcSSHRunner) Run(args ...string) (string, string, error) {
	command := fmt.Sprintf("r --kubeconfig /opt/kubeconfig %s", strings.Join(args, " "))
	stdout, err := r.Runner.Run(command)
	return stdout, "", err
}

func (r OcSSHRunner) RunPrivate(args ...string) (string, string, error) {
	command := fmt.Sprintf("r --kubeconfig /opt/kubeconfig %s", strings.Join(args, " "))
	stdout, err := r.Runner.RunPrivate(command)
	return stdout, "", err
}

func UseOCWithSSH(ssh *ssh.SSHRunner) OcConfig {
	return OcConfig{
		Runner: OcSSHRunner{
			Runner: ssh,
		},
		Context: constants.DefaultContext,
		Cluster: constants.DefaultName,
	}
}
```